_includes/v22.2/metric-names.md

```diff
@@ -90,14 +90,26 @@ Name | Help
 `queue.replicagc.processingnanos` | Nanoseconds spent processing replicas in the replica GC queue
 `queue.replicagc.removereplica` | Number of replica removals attempted by the replica gc queue
 `queue.replicate.addreplica` | Number of replica additions attempted by the replicate queue
+`queue.replicate.addreplica.error` | Number of failed replica additions processed by the replicate queue
+`queue.replicate.addreplica.success` | Number of successful replica additions processed by the replicate queue
 `queue.replicate.pending` | Number of pending replicas in the replicate queue
 `queue.replicate.process.failure` | Number of replicas which failed processing in the replicate queue
 `queue.replicate.process.success` | Number of replicas successfully processed by the replicate queue
 `queue.replicate.processingnanos` | Nanoseconds spent processing replicas in the replicate queue
 `queue.replicate.purgatory` | Number of replicas in the replicate queue's purgatory, awaiting allocation options
 `queue.replicate.rebalancereplica` | Number of replica rebalancer-initiated additions attempted by the replicate queue
 `queue.replicate.removedeadreplica` | Number of dead replica removals attempted by the replicate queue (typically in response to a node outage)
+`queue.replicate.removedeadreplica.error` | Number of failed dead replica replica removals processed by the replicate queue
+`queue.replicate.removedeadreplica.success` | Number of successful dead replica replica removals processed by the replicate queue
+`queue.replicate.removedecommissioningreplica.error` | Number of failed decommissioning replica replica removals processed by the replicate queue
+`queue.replicate.removedecommissioningreplica.success` | Number of successful decommissioning replica replica removals processed by the replicate queue
```

Comment:
perhaps drop the double replica here? and on the other ones cc @AlexTalks

Reply:
Ah, these metrics descriptions are populated verbatim from upstream (you: engineering). I have raised the issue here: https://github.com/cockroachdb/cockroach/issues/92925

I have manually corrected these here, for this instance. PTAL, thank you.